go/fiber/config.yaml

```diff
@@ -0,0 +1,3 @@
+framework:
+  website: github.com/fenny/fiber
```

Comment:
rather use `website: fenny.github.io/fiber/#/` no ? to link directly the documentation from `here`

or `github: fenny/fiber` if you prefer to link github directly